src/components/enterprise-page/data/hooks.js

```diff
@@ -25,6 +26,7 @@ const defaultBrandingConfig = {
  */
 export function useEnterpriseCustomerConfig(enterpriseSlug) {
   const [enterpriseConfig, setEnterpriseConfig] = useState();
+  let fetchError = null;
```

Comment:
If we expect `fetchError` to change, we should put it into component state such that when we set `fetchError` the component will properly re-render and respond to this new state.

As is, we're re-instantiating the `fetchError` variable each time a consuming component of the `useEnterpriseCustomerConfig` hook renders, changing its value in memory but has the potential to get out of sync with the rendering view.

Recommendation: 

```jsx
const [fetchError, setFetchError] = useState();
// ...
setFetchError(new Error('Error: learner portal is not enabled'));
```